From 887aea0af64600d038dc8a9ec7c3c11d34b5429a Mon Sep 17 00:00:00 2001
From: tobw <tobw@users.noreply.github.com>
Date: Wed, 27 May 2015 12:03:07 +0200
Subject: [PATCH 5/6] set global default settings

- set localization
- enable upnp support
- enable webserver support
- enable airplay support
---
 system/settings/settings.xml | 23 ++++++++++++-----------
 1 file changed, 12 insertions(+), 11 deletions(-)

diff --git a/system/settings/settings.xml b/system/settings/settings.xml
index 367542d..deb198b 100644
--- a/system/settings/settings.xml
+++ b/system/settings/settings.xml
@@ -114,7 +114,7 @@
       <group id="1">
         <setting id="locale.language" type="string" label="248" help="36114">
           <level>0</level>
-          <default>English</default>
+          <default>German</default>
           <constraints>
             <options>languages</options>
           </constraints>
@@ -122,7 +122,7 @@
         </setting>
         <setting id="locale.country" type="string" label="20026" help="36115">
           <level>0</level>
-          <default>USA (12h)</default>
+          <default>Deutschland</default>
           <constraints>
             <options>regions</options>
           </constraints>
@@ -144,7 +144,7 @@
         </setting>
         <setting id="locale.keyboardlayouts" type="list[string]" label="310" help="36432">
           <level>1</level>
-          <default>English QWERTY</default>
+          <default>German QWERTZ</default>
           <constraints>
             <options>keyboardlayouts</options>
             <delimiter>|</delimiter>
@@ -232,7 +232,7 @@
       <group id="1">
         <setting id="screensaver.mode" type="addon" label="356" help="36130">
           <level>0</level>
-          <default>screensaver.xbmc.builtin.dim</default>
+          <default></default>
           <constraints>
             <addontype>xbmc.ui.screensaver</addontype>
             <allowempty>true</allowempty>
@@ -828,7 +828,7 @@
       <group id="1">
         <setting id="subtitles.languages" type="list[string]" label="24111" help="24112">
           <level>1</level>
-          <default>English</default>
+          <default>German</default>
           <constraints>
             <options>languages</options>
             <delimiter>,</delimiter>
@@ -1974,7 +1974,7 @@
       <group id="1">
         <setting id="services.devicename" type="string" label="1271" help="36321">
           <level>0</level>
-          <default>Kodi</default>
+          <default>HomePilot2</default>
           <control type="edit" format="string" />
         </setting>
       </group>
@@ -1984,7 +1984,7 @@
       <group id="1">
         <setting id="services.upnpserver" type="boolean" label="21360" help="36323">
           <level>0</level>
-          <default>false</default>
+          <default>true</default>
           <control type="toggle" />
         </setting>
         <setting id="services.upnpannounce" type="boolean" parent="services.upnpserver" label="20188" help="36324">
@@ -2015,7 +2015,7 @@
       <group id="1">
         <setting id="services.webserver" type="boolean" label="263" help="36328">
           <level>1</level>
-          <default>false</default>
+          <default>true</default>
           <control type="toggle" />
         </setting>
         <setting id="services.webserverport" type="integer" parent="services.webserver" label="730" help="36329">
@@ -2030,7 +2030,7 @@
         </setting>
         <setting id="services.webserverusername" type="string" parent="services.webserver" label="1048" help="36330">
           <level>1</level>
-          <default>kodi</default>
+          <default>HomePilot2</default>
           <constraints>
             <allowempty>true</allowempty>
           </constraints>
@@ -2122,7 +2122,7 @@
         </setting>
         <setting id="services.esallinterfaces" type="boolean" label="794" help="36338">
           <level>1</level>
-          <default>false</default>
+          <default>true</default>
           <dependencies>
             <dependency type="enable" setting="services.esenabled">true</dependency>
           </dependencies>
@@ -2173,7 +2173,7 @@
       <group id="1">
         <setting id="services.airplay" type="boolean" label="1270" help="36343">
           <level>1</level>
-          <default>false</default>
+          <default>true</default>
           <control type="toggle" />
         </setting>
         <setting id="services.airplayvolumecontrol" type="boolean" parent="services.airplay" label="1269" help="36541">
@@ -2744,6 +2744,7 @@
           <control type="toggle" />
         </setting>
       </group>
+      <visible>false</visible>
     </category>
     <category id="debug" label="14092" help="36391">
       <group id="1">
-- 
1.9.1

